README.md

```diff
@@ -1,7 +1,8 @@
 # Probabilistic Numerics in Python
 
-[![Build Status](https://travis-ci.org/probabilistic-numerics/probnum.svg?branch=master)](https://travis-ci.org/probabilistic-numerics/probnum)
+[![Build Status](https://img.shields.io/travis/probabilistic-numerics/probnum/master?logo=travis%20ci&logoColor=white&label=Travis%20CI)](https://travis-ci.org/probabilistic-numerics/probnum)
 [![Coverage Status](http://codecov.io/github/probabilistic-numerics/probnum/coverage.svg?branch=master)](http://codecov.io/github/probabilistic-numerics/probnum?branch=master)
+[![Documentation](https://img.shields.io/readthedocs/probnum.svg?logo=read%20the%20docs&logoColor=white&label=Docs)](https://probnum.readthedocs.io)
```

Comment:
Should we have two banners, one for the "default" docs and one for "development"? (or "stable" and "latest", or whatever we decide to use)

Reply:
In light of a merger into `master` in the near future, I would vote for simplicity here and just have a link to `latest` (i.e. `master`). This also seems to be the standard in similar repos.

Reply:
Sounds good :)